Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2.2.3-slim
+MAINTAINER "govuk-role-platform-accounts-members@digital.cabinet-office.gov.uk"
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
```

Comment:
One thing we've found for the publishing platform is that splitting out the "update & upgrade" step and the "install packages" step into two separate `RUN`s means the upgraded Docker image can be cached and used as a base by other images.